src/components/blogCard/BlogCard.js

```diff
@@ -2,14 +2,21 @@ import React from "react";
 import "./BlogCard.css";
 
 export default function BlogCard({ blog }) {
+  function openUrlInNewTab(url) {
+    if(url !== undefined) {
+      var win = window.open(url, "_blank");
+      win.focus();
+    }
+  }
+
   return (
     <div>
       <div class="blog-container">
         <a class="blog-card" href="#blog" >
           <h3 className="blog-title">{blog.title}</h3>
           <p class="small">{blog.description}</p>
           <div class="go-corner" >
-            <div class="go-arrow">→</div>
+            <div class="go-arrow" onClick={() => openUrlInNewTab(blog.url)}>→</div>
```

Comment:
Good Issue. Please add onClick Blog div, not on the arrow.

Reply:
@saadpasta Sure, Done